addons/binding/org.openhab.binding.nest/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nest" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	
+	<bridge-type id="bridge">
+		<label>Nest API Source</label>
+		<description>Provides the layer to talk to the nest apis</description>
+		<config-description>
+		    <parameter-group name="nest">
+			    <advanced>false</advanced>
+			    <description>The pieces for talking to nest</description>
+			    <label>Nest Connection</label>
+			    <context></context>
+		    </parameter-group>
+            <parameter-group name="binding">
+                <advanced>false</advanced>
+                <description>Local settings</description>
+                <label>Binding settings</label>
+                <context></context>
+            </parameter-group>
+            
+			<parameter name="clientId" type="text" groupName="nest">
+				<label>Client id</label>
+				<description>The client id from the nest product page</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="clientSecret" type="text" groupName="nest">
+				<label>Client secret</label>
+				<description>The client secret from the nest product page</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="pincode" type="text" groupName="nest">
+				<label>Pincode</label>
+				<description>The pincode to use after clicking on the authorization url</description>
+				<required>true</required>
+			</parameter>
+            <parameter name="accessToken" type="text" groupName="nest">
```

Comment:
Please reformat this file using the provided code formatter or clean up settings,